src/components/cc-ssh-key-list/cc-ssh-key-list.js

```diff
@@ -8,17 +9,18 @@ import { css, html, LitElement } from 'lit';
 import { classMap } from 'lit/directives/class-map.js';
 import { createRef, ref } from 'lit/directives/ref.js';
 import { repeat } from 'lit/directives/repeat.js';
+import {
+  IconRemixAddCircleFill as IconAdd,
+  IconRemixDeleteBin_5Fill as IconBin,
+  IconRemixKeyFill as IconKey,
+} from '../../assets/cc-remix.icons.js';
```

Comment:
Do you think we'll always alias them so it's easier to understand in the template like this?

I think I like that but we should update the example to reflect this.

Reply:
Self-memo: update how we import icons and alias usage in `cc-example-component.js`